docs/en/using-status/messages-faqs.md

```diff
@@ -7,3 +7,76 @@ hide:
 ---
 
 # Messages FAQs
+
+Messaging is a critical component of the Status app, along with [Status Communities][how-to-use-communities-your-quick-start-guide], the [Status Wallet][how-to-use-status-wallet-your-quick-start-guide] and the [Status Web3 browser][how-to-use-status-browser-your-quick-start-guide]. Status messaging offers individuals and communities uncensored, private and secure communication.
+
+The Status app provides a privacy-centric messenger using decentralized [peer-to-peer (P2P) technology][about-status-messages] and strong end-to-end (E2EE) encryption.
+
+## General questions
+
+### What makes Status messaging decentralized?
+
+Unlike centralized networks, which rely on servers controlled by corporations or governments, Status uses a distributed network of nodes. This network is based on Waku, a [peer-to-peer protocol][about-status-messages] for private, secure and censorship-resistant communication.
+
+### Peer-to-peer networks send messages to every node. Is this not a risk?
+
+No, it's not. While it's true that messages are sent to every node in a peer-to-peer network, these messages are encrypted. Only the intended recipient can decrypt and read the message.
+
+### How is Status messaging censorship-resistant?
+
+The Status peer-to-peer network ensures the resilience of the messaging network. For instance, when a network node is shut down or blocked, you can still connect to other nodes.
+
+Status supports free speech infrastructure that prevents us, or anyone else, from censoring you.
+
+!!! note
+    Nodes in the network still rely on having an internet connection. So if your internet provider (or an authoritarian regime) shuts down your network, Status won't work.
+
+## Status messaging
+
+### Can Status read my messages?
+
+No. No one besides you and the intended recipient can read your messages.
+
+### Are my messages in the blockchain?
+
+No, your messages are not in the blockchain and are not transported through the [Ethereum network][about-the-ethereum-blockchain]. Messages are temporarily stored in the peer-to-peer network.
+
+### Can I delete my messages?
+
+Yes, you can always [edit and delete your messages][edit-and-delete-your-messages].
```

Comment:
It depends on what you mean, we can instruct other users to delete their messages, but it's not a true deletion, since we have no control over store nodes/other users

Reply:
True. We have an article explaining that in more detail ("[Edit and delete your messages](https://help.status.im/en/using-status/edit-and-delete-your-messages/)") so I'll cross-linking the article here. Thanks.